sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,56 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController:
+    """
+    A PID Controller class that uses Parallel to combine the P, I, and D components.
+
+    Parameters
+    ==========
+    KP : Expr, Number
+        Proportional gain.
+    KI : Expr, Number
+        Integral gain.
+    KD : Expr, Number
+        Derivative gain.
+    s : Symbol
+        The complex frequency variable.
+
+    Examples
+    ========
+    >>> from sympy.abc import s
+    >>> from sympy.physics.control.lti import PIDController
+    >>> KP, KI, KD = 1, 0.1, 0.01
+    >>> pid = PIDController(KP, KI, KD, s)
+    >>> print(pid.to_PID_expr())
+    TransferFunction(s*(0.01*s + 1) + 0.1, s, s)
+    """
+    def __init__(self, KP, KI, KD, var):
```

Comment:
I understand your comment on default values, but can you give an example here. What would the coefficient be ?

Reply:
@abhiphile

Reply:
The default value of coefficients for PI is KD = 0 because the resultant transfer function will become `KP + KI/S`.
 For the `PD` controller `, KP+S*KD` is the resultant transfer function with KI = 0, and for the `P` type controller, we have transfer function `Kp` having `Kp = 0` and `Kd = 0`. So the default value will be  Zero here for all.

Reply:
Okay I get your suggestion. This must be handled. 

Maybe instead of directly giving 0 as default coefficient we can have something like this for flexibility-
```
def __init__(self, KP=None, KI=None, KD=None, var):
    if KP is None: KP = 0
    if KI is None: KI = 0
    if KD is None: KD = 0
    // check if the inputs are symbolic expressions or numbers in the first place
```
But this can be looked into later.